### PR TITLE
Make colors for challenge difficulty more easy to change

### PR DESCRIPTION
**Pull request by Rand0mLetters**
some colors got overwritten but now beginner difficulty is light blue, easy is green, medium is yellow, hard is light red, extreme is a very menacing red.

also made it easier to change the colors (now in styles/globals.css)

### CODE REVIEW

**Comment by Hacksore** [comment]
@Rand0mLetters can you include some screenshots for us reviewers.

**Comment by Rand0mLetters** [comment]
oh yeah for sure

I'm busy right now, I'll get them in the next 3-5h?

File: apps/web/src/app/explore/_components/explore-card.tsx [hunk 1]
```diff
@@ -20,24 +20,24 @@ interface ExploreCardProps {
...
   BEGINNER:
-    'dark:hover:border-pink-300 hover:border-pink-500 dark:group-focus:border-pink-300 group-focus:border-pink-500',
-  EASY: 'dark:hover:border-green-300 hover:border-green-500 dark:group-focus:border-green-300 group-focus:border-green-500',
+    'dark:hover:border-difficultyBeginnerDark hover:border-difficultyBeginner dark:group-focus:border-difficultyBeginnerDark group-focus:border-difficultyBeginner',
+  EASY: 'dark:hover:border-difficultyEasyDark hover:border-difficultyEasy dark:group-focus:border-difficultyEasyDark group-focus:border-difficultyEasy',
   MEDIUM:
-    'dark:hover:border-yellow-300 hover:border-yellow-500 dark:group-focus:border-yellow-300 group-focus:border-yellow-500',
-  HARD: 'dark:hover:border-red-300 hover:border-red-500 dark:group-focus:border-red-300 group-focus:border-red-500',
+    'dark:hover:border-difficultyMediumDark hover:border-difficultyMedium dark:group-focus:border-difficultyMediumDark group-focus:border-difficultyMedium',
+  HARD: 'dark:hover:border-difficultyHardDark hover:border-difficultyHard dark:group-focus:border-difficultyHardDark group-focus:border-difficultyHard',
```
Comment (bautistaaa) [inline apps/web/src/app/explore/_components/explore-card.tsx:27]: kebab case these

**Comment by Rand0mLetters** [comment]
image
image
image
image
image
image
and in light mode (colors can be adjusted):
image
image
image
image
image

going to work on camelcasing the names in about 2 hours (my time estimation is really bad though, so it might be earlier)

**Review by PickleNik — CHANGES_REQUESTED** [review]
Bad contrast ratios. Please use at least -600 for light mode and -400 or better -300 for dark

**Comment by PickleNik** [comment]
Mainly light blue and light red in light mode are off.
And green and yellow don't look good on dark.

But I like the change of colors, although I'd rethink Hard and Extreme, maybe Orange (or keep it yellow as it was) for Mediums, Red as it was for Hards, and Purple/Indigo for Extremes?

**Comment by Rand0mLetters** [comment]
I've updated the colors and converted them to hsl
They pass WCAG (except the light mode colors, on WCAG AAA with normal text, but it should(?) be fine, I think?)

The dark colors look very pastel-y, is that ok?

image
image
image
image
image


Dark mode (couldn't figure out the yellow color):
image
image
image
image
image

**Comment by Rand0mLetters** [comment]
Oh shoot, have to update the colors on /tracks. Don't merge yet please

**Comment by PickleNik** [comment]
this looks great, I would also make `HARD` and `MEDIUM` just 1 tone lighter on Dark mode, and `MEDIUM` one tone lighter on Light mode as well

**Review by PickleNik — APPROVED** [review]
do whatever u need to do for tracks but LGTM 👍🏼

**Comment by Rand0mLetters** [comment]
just finished, running checks and pushing soon
spent too long figuring out how to use tailwind alpha values

**Comment by Rand0mLetters** [comment]
image
image

ready for merge (I think) @PickleNik

**Pull request merged**